src/components/molecules/PoolTransactions.tsx

```diff
@@ -11,7 +11,7 @@ import { formatPrice } from '../atoms/Price/PriceUnit'
 import { gql, useQuery } from '@apollo/client'
 import {
   TransactionHistory,
-  TransactionHistoryPoolTransactions
+  TransactionHistory_poolTransactions as TransactionHistoryPoolTransactions
```

Comment:
ran into this weirdness too, the Apollo typings generated locally and the the ones generated in CI differ for whatever reason. This is also why Travis fails https://travis-ci.com/github/oceanprotocol/market/builds/215211030#L579

I ended up leaving this as is (`TransactionHistoryPoolTransactions `) and learned to ignore the error in my editor 🙃

Reply:
well locally this change fixed the error, but in github Travis test seems to fail either way

Reply:
if you get that locally it means something after `find` isn't working for you locally
https://github.com/oceanprotocol/market/blob/a76093556495444b0f6735e0fe52211e5fcc8ec9/package.json#L22

Reply:
gonna investigate this cause with the new GiHub Actions setup I am able to actually capture this, this fails on macOS https://github.com/oceanprotocol/market/runs/1806022553?check_suite_focus=true

Reply:
something strange its happening when I try to resolve the conflicts locally. I have conflicts in 24 files, and none of them was changed by me